### PR TITLE
update distributor_config docs to be clearer

### PR DESCRIPTION
**Pull request by asmacdo**
https://pulp.plan.io/issues/1362

### CODE REVIEW

File: docs/dev-guide/integration/rest-api/repo/cud.rst [hunk 1]
```diff
@@ -87,7 +87,8 @@ is centered around updating only that metadata.
 
 * :param:`?delta,object,object containing keys with values that should be updated on the repository`
 * :param:`?importer_config,object,object containing keys with values that should be updated on the repository's importer config`
-* :param:`?distributor_configs,object,object containing keys that are distributor ids, and values that are objects containing keys with values that should be updated on the specified distributor's config`
+* :param:`?distributor_configs,object,dict containing keys that are distributor ids, and values that are dictionaries containing plugin specific keys/value pairs`
```
Comment (rbarlow) [inline docs/dev-guide/integration/rest-api/repo/cud.rst]: In JSON there is no such thing as a dict - it's called an object.

http://json.org/

File: docs/dev-guide/integration/rest-api/repo/cud.rst [hunk 2]
```diff
@@ -87,7 +87,8 @@ is centered around updating only that metadata.
...
+* :param:`?delta,object, dictionary containing keys with values that will update the distributor object, currently only supports "auto_publish"`
```
Comment (rbarlow) [inline docs/dev-guide/integration/rest-api/repo/cud.rst]: Same here, it's an object not a dictionary.

File: docs/dev-guide/integration/rest-api/repo/cud.rst [hunk 3]
```diff
@@ -400,6 +402,10 @@ See :ref:`bind` for details.
 | :method:`put`
 | :path:`/v2/repositories/<repo_id>/distributors/<distributor_id>/`
 | :permission:`update`
+| :param_list:`put`
+
+* :param:`?distributor_config,object,dictionary containing plugin specific keys with values that will update the distributor config`
```
Comment (rbarlow) [inline docs/dev-guide/integration/rest-api/repo/cud.rst]: s/dictionary/object/

File: docs/dev-guide/integration/rest-api/repo/cud.rst [hunk 4]
```diff
@@ -400,6 +402,10 @@ See :ref:`bind` for details.
...
+* :param:`?delta,object, dictionary containing keys with values that will update the distributor object, currently only supports "auto_publish"`
```
Comment (rbarlow) [inline docs/dev-guide/integration/rest-api/repo/cud.rst]: s/dictionary/object/

**Comment by rbarlow** [comment]
LGTM, just make sure to remove all the uses of dictionary here. Dictionary is a Python-specific terminology, and our users are sending us JSON over REST and don't necessarily know what dictionaries are. The REST docs should only discuss HTTP/JSON/REST concepts.

**Comment by asmacdo** [comment]
ok test

**Pull request merged**